src/renderer/src/types/index.ts

```diff
@@ -836,6 +836,7 @@ export type S3Config = {
   autoSync: boolean
   syncInterval: number
   maxBackups: number
+  skipTlsVerify: boolean
```

Comment:
这里应该设为可选值吧，然后默认为false即可
```suggestion
  skipTlsVerify?: boolean
```